omnibus/files/private-chef-cookbooks/private-chef/spec/libraries/helper_spec.rb

```diff
@@ -113,7 +113,40 @@
       end
     end
 
+    shared_context "elastic search version request fails" do
```

Comment:
I copied the idea from [here](https://github.com/chef/chef-server/blob/608dbe94d15822a31849952e13549744fc40a702/oc-chef-pedant/spec/api/versioned_behaviors/server_api_v1_spec.rb#L90). 

I did ensure that the test is working as intended by changing the `times` variable to 7 and watching it fail.